src/components/feature/auth/signup/sign-up-basic/container.ts

```diff
@@ -1,16 +1,19 @@
 import { useDispatch } from "react-redux";
+
 import { auth } from "@components/feature/firebase";
+
 import { stateStepsControl } from "@actions/index";
 
+import { toast } from "react-toastify";
+
 export const useSingUpBasicContainer = (): any => {
   const dispatch = useDispatch();
   const createAccount = async ({ email, password }) => {
-    console.log("next clicked");
     try {
       await auth.createUserWithEmailAndPassword(email, password);
       await dispatch(stateStepsControl.updateSteps(1));
     } catch (error) {
-      console.log(error);
+      toast.error(error);
```

Comment:
to Ci raczej nie zadziała? :D mi się wydaje, że musisz to drillnąć, chyba, że firebase obsługuje to przez po prostu error?